_tour/traits.md

```diff
@@ -36,6 +34,12 @@ Extending the `trait Iterator[A]` requires a type `A` and implementations of the
 ## Using traits
 Use the `extends` keyword to extend a trait. Then implement any abstract members of the trait using the `override` keyword:
 ```tut
+trait Iterator[A] {
+  def hasNext: Boolean
+  def next(): A
+}
+
```

Comment:
Could we remove this extra space? :smile:

Reply:
I assume by extra space you mean line #41.

Reply:
or extra space in 38,39.